connectors/citrus-openapi/src/test/java/org/citrusframework/openapi/actions/OpenApiClientActionBuilderTest.java

```diff
@@ -1,25 +1,23 @@
 package org.citrusframework.openapi.actions;
```

Comment:
some more tests with missing copyright headers. there's this function in Intellij, where you can apply copyright headers to all files matching a glob pattern (e.g. `**/*.java`). maybe you should try it 😉 or I can do it, if you want me to.

Reply:
@christophd fixed this. thanks!